### PR TITLE
Remove the tooltip for HVDC lines

### PR DESCRIPTION
**Pull request by EstherDarkish**

### CODE REVIEW

File: src/components/network/network-map.js [hunk 1]
```diff
@@ -410,11 +412,14 @@ const NetworkMap = (props) => {
                 labelSize: LABEL_SIZE,
                 pickable: true,
                 onHover: ({ object, x, y }) => {
+                    let equipmentId = undefined;
                     if (object) {
+                        equipmentId = getNameOrId(object?.line ?? object);
```
Comment (ghelloumi) [inline src/components/network/network-map.js:417]:
```suggestion
                       const equipmentId = object && getNameOrId(object?.line ?? object);
```

File: src/components/network/network-map.js [hunk 2]
```diff
@@ -410,11 +412,14 @@ const NetworkMap = (props) => {
...
+                    }
+                    if (!!equipmentId) {
                         setCursorType('pointer');
-                        const lineObject = object?.line ?? object;
                         setTooltip({
-                            equipmentId: getNameOrId(lineObject),
+                            equipmentId: equipmentId,
```
Comment (ghelloumi) [inline src/components/network/network-map.js:422]:
```suggestion
                            equipmentId,
```

File: src/components/network/network-map.js [hunk 3]
```diff
@@ -253,7 +253,9 @@ const NetworkMap = (props) => {
     function renderTooltip() {
         return (
             tooltip &&
-            tooltip.visible && (
+            tooltip.visible &&
```
Comment (ghelloumi) [inline src/components/network/network-map.js:256]:
```suggestion
            tooltip?.visible &&
```

then remove tooltip && (line 255)

File: src/components/network/network-map.js [hunk 4]
```diff
@@ -253,7 +253,9 @@ const NetworkMap = (props) => {
...
+            // Should not display the tooltip for HVDC lines
+            !props.mapEquipments.getHvdcLine(tooltip.equipmentId) && (
```
Comment (ghelloumi) [inline src/components/network/network-map.js:258]: can you define mapEquipments in propTypes please? and if it is not required you can add mapEquipments."?" for security thank you

File: src/components/network/network-map.js (hunk 2 above)
Comment (EstherDarkish) [reply src/components/network/network-map.js:422]: Done

File: src/components/network/network-map.js (hunk 3 above)
Comment (EstherDarkish) [reply src/components/network/network-map.js:256]: Done

File: src/components/network/network-map.js (hunk 4 above)
Comment (EstherDarkish) [reply src/components/network/network-map.js]: Done

**Review by ghelloumi — APPROVED** [review]

File: src/components/network/network-map.js [hunk 5]
```diff
@@ -252,8 +252,9 @@ const NetworkMap = (props) => {
 
     function renderTooltip() {
         return (
-            tooltip &&
-            tooltip.visible && (
+            tooltip?.visible &&
```
Comment (souissimai) [inline src/components/network/network-map.js:255]: suggestion : 
    const shouldRenderTooltip =
        tooltip?.visible &&
        !props.mapEquipments?.getHvdcLine(tooltip.equipmentId);

**Review by souissimai — APPROVED** [review]
test OK, code Ok

File: src/components/network/network-map.js (hunk 5 above)
Comment (EstherDarkish) [reply src/components/network/network-map.js:255]: Thanks for the suggestion, but I won't do a refactorization in this US.

**Pull request merged**